ors-api/ors-config.yml

```diff
@@ -12,17 +12,17 @@ ors:
 #    allowed_headers: Content-Type, X-Requested-With, accept, Origin, Access-Control-Request-Method, Access-Control-Request-Headers, Authorization
 #    preflight_max_age: 600
   engine:
+    graphs_root_path: ./ors-core/data/graphs
```

Comment:
Is there any specific reason why the default is to put the graphs under `./ors-core/data/graphs` rather than having them under `./graphs`? 🤔

Reply:
Yes, it reflects the setup used within the docker container. Personally I think that's something we could get rid of when we redo the docker setup after we merge the jar deployment...